x-pack/plugins/uptime/public/routes.tsx

```diff
@@ -153,20 +153,17 @@ export const PageRouter: FC = () => {
       flex-wrap: wrap;
     }
   `;
+  const { http } = useKibana().services;
+  const basePath = http!.basePath.get();
```

Comment:
@snide Can you confirm on your end if this works? When I go to Uptime pages, it just completely hangs for me.

Reply:
I mean actually getting to that page in Uptime. Whenever I try to reach `/app/uptime` it just hangs. If I comment out those two lines, the page loads. I'll keep testing though.